django/db/migrations/autodetector.py

```diff
@@ -63,6 +64,8 @@ def deep_deconstruct(self, obj):
                 key: self.deep_deconstruct(value)
                 for key, value in obj.items()
             }
+        elif isinstance(obj, functools.partial):
+            return (obj.func, self.deep_deconstruct(obj.args), self.deep_deconstruct(obj.keywords))
```

Comment:
Could you expand the tests -- the existing one passes with simply `return obj.func` here. It would be good to have tests to show why the rest of the tuple is needed.